README.md

```diff
@@ -72,11 +72,11 @@ If you brillantly completed those two steps, you should already be able to play
 ### III) Listen and react to events
 The final part of this workshop is to implement the `interactive` command. This command should allow you to play a whole game interactively, prompting you for your next move when it's your turn to play and showing your opponent's moves in real time.
 
-To achieve this you will have to subscribe to contract events. Those events could be found
+To achieve this you will have to subscribe to contract events. Those events could be found [here](https://mumbai.polygonscan.com/address/0x20deb5dbdee348d3c53b63c8e7e2263502dd5618#code#F1#L10)
```

Comment:
TIL that you can use target specific contract file and specific contract line using URL query.